### ISSUE TITLE
Make "About" availble directly from the menu bar icon

### ISSUE THREAD

**Issue by mfn** [lwouis/alt-tab-macos#574]
Since I'm a bit active comparing problems with versions and reporting issues, I sometimes have to remind myself on which version I'm. Everytime I open the menu I stumbled where to find "About" because in my experience, the majority of application allow direct access in the menu bar.

AltTab:
![image](https://user-images.githubusercontent.com/87493/92205179-6e7d7c00-ee85-11ea-87a6-67c5eabe4e69.png)

Spectable:
![image](https://user-images.githubusercontent.com/87493/92205214-7f2df200-ee85-11ea-9e61-d807162b561d.png)

f.lux:
![image](https://user-images.githubusercontent.com/87493/92205255-8fde6800-ee85-11ea-9138-364917f8f1e3.png)

Of course there are others also _not_ having it.

But from my DX perspective, it's nicer to have it directly there.

Thanks!

**Comment by lwouis** [comment]
Hi @mfn! Funny you asked that!

This was originally done, for DX reasons as you said, back in 2019 in the early days (https://github.com/lwouis/alt-tab-macos/issues/36). It was then moved to the "Quit" button which was showing something like "Quit AltTab 2.0.1" (de7f428ba15481b4948c82c4f7108ab8ae1486e5). It was finally removed and migrated to the preference windows (291f872d3dfaecd83013a4d1c92dbab222a2f89c)

Let's put it back 👍

I checked the Apple guidelines, and they actually have [specifics](https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-bar-menus/) about this:

![image](https://user-images.githubusercontent.com/106195/92207364-0ef29100-eec4-11ea-9a6d-afcdf1e42117.png)

> Don’t include a version number with your app name. Version information belongs in the About window, not in your app’s menus.

> Separate the About menu item and display it first. Include a separator after the About menu item and don’t group it with other items.

**Comment by lwouis** [comment]
I implemented this. It will be in the next release

**Comment by SoPat712** [comment]
I personally think it should just say about, I'm not sure what the guidelines are on that, though. My apps are split on whether they say it or not.

MacDroid, an app i use frequently doesn't use it.

![image](https://user-images.githubusercontent.com/30350506/92271172-532a6500-eeb5-11ea-9eb5-c733f27202b5.png)


While Mountain Duck, another app I use frequently, doesn't.

![image](https://user-images.githubusercontent.com/30350506/92271212-65a49e80-eeb5-11ea-8d03-ba8860555ba3.png)

**Comment by lwouis** [comment]
I just tested like 5 or 6 Apple apps and they all add the name. I'm betting they all do. They have guidelines + they actually do it themselves = I'll do it too. Let's keep the UX consistent for the user.

**Comment by SoPat712** [comment]
Yeah I guess that makes sense, to keep with the flow of other apps!
